setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
```

Comment:
This file shouldn't be here anymore, since it's been superseded by the pyproject.toml file

Reply:
I found this was necessary to run "python setup.py install", though perhaps that's the wrong way to install things now? If you need it removed for the pull I'll remove it.

Reply:
If you're trying to install from your local repo to another environment, you _should_ be able to use `pip install /path/to/local/repo`.  If that _doesn't_ work, that probably means there's something wrong with the pyproject.toml file, so please let me know if that's the case.

Reply:
That does work, I've made the change. But to be sure it's what you want: this means that others that try "python setup.py install" will find that it doesn't work.